### PR TITLE
release notes update for 6.9.0-6863

### PR DESCRIPTION
**Pull request by lspxxv**
## Description

Added the new features, improvements and fixes for the 6.9.0 release for the sensu docs 

## Motivation and Context

Updates the release notes to include information for the 6.9.0 release

### CODE REVIEW

File: content/sensu-go/6.9/release-notes.md [hunk 1]
```diff
@@ -111,6 +112,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.9.0 release notes
+​
+**November 1, 2022** &mdash; The latest release of Sensu Go, version 6.9.0, is now available for download.
+​
+Sensu Go 6.9.0 provides a mix of new features as well as customer experience improvements and bug fixes. The new CyberArk integration provides a new way to store your secrets, while we also now support metric tags in the Graphite plaintext protocol. Additionally, the agent websocket connection log entries now include the backend name and authentication requests are now logged with their status. Under the hood, the keepalive reconstructions no longer block backend startup and now run gradually in the background and etcd was bumped to 3.5.5. Lastly, a number of usability enhancements and bug fixes were implemented for the Web UI .
```
Comment (hillaryfraley) [inline content/sensu-go/6.9/release-notes.md]:
```suggestion
Sensu Go 6.9.0 provides a mix of new features, customer experience improvements, and bug fixes. The new CyberArk integration provides a new way to store your secrets, and Sensu also now supports metric tags in the Graphite plaintext protocol. Additionally, agent websocket connection log entries now include the backend name and authentication requests are now logged with their status. Under the hood, keepalive reconstructions no longer block backend startup &mdash; they now run gradually in the background &mdash; and we bumped the etcd version to 3.5.5. Last but not least, Sensu Go 6.9.0 implements a number of usability enhancements and bug fixes for the web UI.
```

File: content/sensu-go/6.9/release-notes.md [hunk 2]
```diff
@@ -111,6 +112,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- ([Commercial feature][303]) In the web UI, the check detail page now indicates when Sensu could not fetch a dynamic runtime asset listed in the check definition.
+- ([Commercial feature][303]) In the web UI, the mutation dialog now properly reports unauthorized errors.
+- ([Commercial feature][303]) In the web UI, saved searches now must have unique names to prevent accidentally overwriting existing saved searches.
+- Fixed an issue that prevented multi-expression, exclusive event filters set to `deny` from being evaluated properly.
+​
+​
+[303]: /sensu-go/6.9/commercial/
+[304]: /sensu-go/6.9/operations/manage-secrets/secrets-providers/#cyberarkprovider-spec-attributes
+[305]: /sensu-go/6.9/operations/manage-secrets/secrets-management/#use-cyberark-conjur-for-secrets-management
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.9/release-notes.md]:
```suggestion
```

Comment (hillaryfraley) [reply content/sensu-go/6.9/release-notes.md]: I moved these link references to the end of the doc (with the other refs)

File: content/sensu-go/6.9/release-notes.md [hunk 3]
```diff
@@ -111,6 +112,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
...
+- ([Commercial feature][303]) In the web UI, the check detail page now indicates when Sensu could not fetch a dynamic runtime asset listed in the check definition.
+- ([Commercial feature][303]) In the web UI, the mutation dialog now properly reports unauthorized errors.
+- ([Commercial feature][303]) In the web UI, saved searches now must have unique names to prevent accidentally overwriting existing saved searches.
+- Fixed an issue that prevented multi-expression, exclusive event filters set to `deny` from being evaluated properly.
+
+
+[303]: /sensu-go/6.9/commercial/
+[304]: /sensu-go/6.9/operations/manage-secrets/secrets-providers/#cyberarkprovider-spec-attributes
+[305]: /sensu-go/6.9/operations/manage-secrets/secrets-management/#use-cyberark-conjur-for-secrets-management
+
```
Comment (hillaryfraley) [inline content/sensu-go/6.9/release-notes.md]:
```suggestion
```

File: content/sensu-go/6.9/release-notes.md [hunk 4]
```diff
@@ -111,6 +112,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.9.0 release notes
+
+**November 1, 2022** &mdash; The latest release of Sensu Go, version 6.9.0, is now available for download.
+
+Sensu Go 6.9.0 provides a mix of new features as well as customer experience improvements and bug fixes. The new CyberArk integration provides a new way to store your secrets, while we also now support metric tags in the Graphite plaintext protocol. Additionally, the agent websocket connection log entries now include the backend name and authentication requests are now logged with their status. Under the hood, the keepalive reconstructions no longer block backend startup and now run gradually in the background and etcd was bumped to 3.5.5. Lastly, a number of usability enhancements and bug fixes were implemented for the Web UI.
```
Comment (hillaryfraley) [inline content/sensu-go/6.9/release-notes.md]:
```suggestion
Sensu Go 6.9.0 provides a mix of new features, customer experience improvements, and bug fixes. The new CyberArk integration provides a new way to store your secrets, and Sensu also now supports metric tags in the Graphite plaintext protocol. Additionally, agent websocket connection log entries now include the backend name and authentication requests are now logged with their status. Under the hood, keepalive reconstructions no longer block backend startup &mdash; they now run gradually in the background &mdash; and we bumped the etcd version to 3.5.5. Last but not least, Sensu Go 6.9.0 implements a number of usability enhancements and bug fixes for the web UI.
```

**Review by hillaryfraley — APPROVED** [review]
🎉

**Review by c-kruse — APPROVED** [review]
Looks good!

**Pull request merged**